pkg/plugins/resources/dockerfile/main.go

```diff
@@ -21,6 +21,9 @@ type Spec struct {
 	Instruction types.Instruction `yaml:"instruction,omitempty"`
 	// Value specifies the value for a specified Dockerfile instruction.
 	Value string `yaml:"value,omitempty"`
+	// Stage specifies the stage to extract the value from, if None specified, the latest stage will be used
+	// only for sources
```

Comment:
@loispostula This is really awesome and I wonder if it would require a lot of effort to support condition/target as well?

Reply:
@olblak I need to update the comment, it's been implemented for all